MAUI/DataGrid/grid-events.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Grid Events in MAUI DataGrid control | Syncfusion
+description: Learn here all about Grid Events support in Syncfusion MAUI DataGrid (SfDataGrid) control and more about it.
```

Comment:
Learn here all about data grid events that help identify interactions on the cell, and listen to cell value changes in the Syncfusion MAUI DataGrid (SfDataGrid) control, and learn more about it

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/2a55c227-58ed-4c54-8a22-b93dcbeafd97)
I had updated as discussed.